### DISCUSSION TITLE
does rememberable option is valid for API-only mode

### DISCUSSION THREAD

**Discussion by mtoygar** [gonzalo-bulnes/simple_token_authentication#380] [Q&A]
according to devise wiki below rememberable is not supported with the API-only mode.

https://github.com/heartcombo/devise/wiki/API-Mode-Compatibility-Guide

However, on the `simple_token_authentication` README, there are examples with `rememberable` option. Does `simple_token_authentication` also provides compatibility with the `rememberable` option. If not, removing it and adding a note might be beneficial.

**Comment by gonzalo-bulnes** [comment]
Good question.

To be clear, Simple Token Authentication doesn't provide any extra support for use cases that are not supported by Devise.

That being said README examples are not meant to be specific to the API-only mode (nor the the full Rails mode by the way, they're generic examples). Because of that I wouldn't remove the mention to `rememberable` or `lockable` etc. but I think it could be beneficial to add a note to clarify that the options available may vary depending on your usage, along with the link you provided above. 

What do you think, does this make sense to you @mtoygar? If so, do you want to add that note to the README?
